src/lib/viewers/BaseViewer.js

```diff
@@ -169,6 +170,8 @@ class BaseViewer extends EventEmitter {
             this.startAt = getProp(this.options, `fileOptions.${this.options.file.id}.${FILE_OPTION_START}`, {});
         }
 
+        this.api = this.options.api || new Api();
```

Comment:
It looks like most other files handle this in their constructor. Is the difference here meaningful?